acceptance-tests/src/test/java/uk/gov/di/test/pages/SupportPage.java

```diff
@@ -10,6 +10,11 @@ public class SupportPage extends SignIn {
     By supportLink = By.xpath("//*[contains(text(), 'Support (opens in new tab)')]");
     By moreDetailsField = By.id("moreDetailDescription");
     By successMessage = By.cssSelector(".govuk-panel--confirmation");
+    // TODO: Need to get the correct ID for the field
+    By securityCodeField = By.id("securityCode");
```

Comment:
Have a look at the Jira board to see if the work for this screen has been done or is underway yet. If it is, just give the assigned dev a shout and get them to give you the id's for the element(s)